VL.Models/Model/Book.cs

```diff
@@ -1,13 +1,15 @@
 ﻿using System.ComponentModel.DataAnnotations;
 
-namespace VirtualLibrary.Models
+namespace VL.Models
 {
     public class Book
     {
-
-        public int ID { get; set; }
+        [Key]
+        public int Id { get; set; }
         public string Title { get; set; } = string.Empty;
+        [Required]
         public string Author { get; set; } = string.Empty;
+        [Required]
 
         [DataType(DataType.Date)]
```

Comment:
This is not required, DateTime is inferred from your model already because you specified it

'DateTime ReleaseDate'